### PR TITLE
iOS player features

### LINKED ISSUE #34: [feature request] Auto change to the next episode when we finished an episode
Auto change to the next episode when we finished an episode

### LINKED ISSUE #40: [Feature Request] Playback speed control (preferably 1.1x, 1.2x, 1.3x.....)
Would love to have playback speed control in the app in increments of 1.1x, 1.2x,1.3x and such.

### PR DESCRIPTION
**Pull request by stephenb10**
- Update the now playing center for iOS
- Add a next up preview for episodes in the video player (fix #34)
- Add playback speed control (fix #40), currently has speed selection from 0.25 to 2.0 with 0.25 increments like YouTube, would be happy to see if there are better increments

Simulator Screen Shot - iPhone 12 - 2021-06-26 at 21 24 43
Simulator Screen Shot - iPhone 12 - 2021-06-26 at 21 24 40

### CODE REVIEW

**Comment by acvigue** [comment]
looks good so far!

File: JellyfinPlayer/VideoPlayerSettingsView.swift [hunk 1]
```diff
@@ -80,6 +95,7 @@ struct VideoPlayerSettings: View {
         .onAppear(perform: {
             _captionTrack.wrappedValue = self.delegate.selectedCaptionTrack
             _audioTrack.wrappedValue = self.delegate.selectedAudioTrack
+            playbackSpeedSelection = self.delegate.selectedPlaybackSpeedIndex
```
Comment (stephenb10) [inline JellyfinPlayer/VideoPlayerSettingsView.swift:98]: Is there any advantage to assigning the value with `_object.wrappedValue`  compared to just assigning the variable directly?

Comment (acvigue) [reply JellyfinPlayer/VideoPlayerSettingsView.swift:98]: No and I prefer just assigning it directly. This is the first time I've used Swift and didn't know that both worked. If you see any, feel free to change them.

**Comment by acvigue** [comment]
I'll work on the design of the Up Next view if you can get it implemented :p

**Pull request merged**